### ISSUE TITLE
Empty popup when changing page layout

### ISSUE THREAD

**Issue by cweiske** [FluidTYPO3/fluidpages#295]
I get an empty message popup when changing the fluidpages page layout with TYPO3 7.6.

The way those messages are built has changed in the core, and the label does not exist anymore:
```
$ ack -i onchange
Classes/Backend/PageLayoutSelector.php
107:		$onChange = 'onclick="if (confirm(TBE_EDITOR.labels.onChangeAlert) && TBE_EDITOR.checkSubmit(-1)){ TBE_EDITOR.submitForm() };"';
121:				$selector .= '<input type="radio" name="' . $name . '" ' . $onChange . '" value="" ' . $selected . '/> ' . $emptyLabel . LF;
```

**Comment by Apen** [comment]
Same problem here, any solution?

**Comment by NamelessCoder** [comment]
The reason the popup now fails is that the method to dispatch the popup is now changed (to some sort of event based structure rather than something we can call on-demand; problem is exactly like @cweiske states). 

Unfortunately, the right solution to *that* issue and others concerning FormEngine will involve migrating the page layout selector to proper TCA fields (they're custom HTML currently) and **THAT** in turn causes a more serious issue: https://forge.typo3.org/issues/76509. The moment that one is solved I've got a solution ready to push and will of course do so. That fix then solves this issue automatically. I would consider the TYPO3 issue a regression and would expect a fix for LTS too.

I'll take the liberty of closing this one although it's a little ahead of time. The reworked template selection fields *will* come, they're merely delayed a bit by the linked issue ;)
